Failure on windows - other issue

I have an issue running the example with static-julia (today's master).
Windows 7, 64 bit, JuliaPro 0.6.0.1, cygwin (latest).

Run from cygwin's Bash:

```julia
$ c:/JuliaPro-0.6.0.1/Julia-0.6.0/bin/julia.exe juliac.jl hello.jl
Program file:
C:\Users\phlavenk\Documents\Python_Scripts\static-julia-master\hello.jl
Make directory:
"C:\Users\phlavenk\Documents\Python_Scripts\static-julia-master\builddir"
Change directory:
"C:\Users\phlavenk\Documents\Python_Scripts\static-julia-master\builddir"
Running command:
`'C:\JuliaPro-0.6.0.1\Julia-0.6.0\bin\julia.exe' -Cx86-64 '-JC:\JuliaPro-0.6.0.1\Julia-0.6.0\lib\julia\sys.dll' --compile=yes --depwarn=yes --startup-file=no --output-o hello.o -e 'include("C:\\Users\\phlavenk\\Documents\\Python_Scripts\\static-julia-master\\hello.jl"); push!(Base.LOAD_CACHE_PATH, "C:\\JuliaPro-0.6.0.1\\pkgs-0.6.0.1\\lib\\v0.6"); empty!(Base.LOAD_CACHE_PATH)'`
WARNING: @ngenerate is deprecated, used @generated function or (preferably) tuples/CartesianIndex instead
Stacktrace:
 [1] depwarn(::String, ::Symbol) at .\deprecated.jl:70

..... many deprecations ommited (not important now)

Running command:
`gcc -m64 -shared -o libhello.dll hello.o -std=gnu99 '-IC:\JuliaPro-0.6.0.1\Julia-0.6.0\include\julia' -DJULIA_ENABLE_THREADING=1 '-LC:\JuliaPro-0.6.0.1\Julia-0.6.0\bin' -Wl,--stack,8388608 -ljulia -lopenlibm '-Wl,-rpath,$ORIGIN' -Wl,--export-all-symbols`
hello.o: In function `julia_try_yieldto_247':
./event.jl:179: undefined reference to `__imp__resetstkoflw'
./event.jl:179:(.text+0x4d24): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189: undefined reference to `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x557e): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189: undefined reference to `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x71fc): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189: undefined reference to `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x7264): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189: undefined reference to `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0xc4cd): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189: more undefined references to `__imp__resetstkoflw' follow
hello.o:./event.jl:189:(.text+0xf735): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0xf9ac): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x10765): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x108d9): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x122da): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `__imp__resetstkoflw'
hello.o:./event.jl:189:(.text+0x12536): additional relocation overflows omitted from the output
collect2: error: ld returned 1 exit status
ERROR: LoadError: failed process: Process(`gcc -m64 -shared -o libhello.dll hello.o -std=gnu99 '-IC:\JuliaPro-0.6.0.1\Julia-0.6.0\include\julia' -DJULIA_ENABLE_THREADING=1 '-LC:\JuliaPro-0.6.0.1\Julia-0.6.0\bin' -Wl,--stack,8388608 -ljulia -lopenlibm '-Wl,-rpath,$ORIGIN' -Wl,--export-all-symbols`, ProcessExited(1)) [1]
Stacktrace:
 [1] pipeline_error(::Base.Process) at .\process.jl:682
 [2] run(::Cmd) at .\process.jl:651
 [3] julia_compile(::String) at C:\Users\phlavenk\Documents\Python_Scripts\static-julia-master\juliac.jl:49
 [4] main() at C:\Users\phlavenk\Documents\Python_Scripts\static-julia-master\juliac.jl:64
 [5] include_from_node1(::String) at .\loading.jl:569
 [6] include(::String) at .\sysimg.jl:14
 [7] process_options(::Base.JLOptions) at .\client.jl:305
 [8] _start() at .\client.jl:371
while loading C:\Users\phlavenk\Documents\Python_Scripts\static-julia-master\juliac.jl, in expression starting on line 67
```
The hello.o file is created in the builddir (~47.5 MB), but then the linking stage is unsuccessful.

Could you please help me to figure out what's the root cause of the error? I see that there are people that managed to compile the example on windows. Am I missing any library dependence? Thank for your ideas.

Petr

--- a/juliac.jl
+++ b/juliac.jl
@@ -40,15 +40,16 @@ function julia_compile(julia_program_file)
     cflags = Base.shell_split(readstring(`$command --cflags`))
     ldflags = Base.shell_split(readstring(`$command --ldflags`))
     ldlibs = Base.shell_split(readstring(`$command --ldlibs`))
+    cc = is_windows() ? "x86_64-w64-mingw32-gcc" : "gcc"
 
-    command = `gcc -m64 -shared -o $SO_FILE $O_FILE $cflags $ldflags $ldlibs -Wl,-rpath,\$ORIGIN`
+    command = `$cc -m64 -shared -o $SO_FILE $O_FILE $cflags $ldflags $ldlibs -Wl,-rpath,\$ORIGIN`
     if is_windows()
         command = `$command -Wl,--export-all-symbols`
     end
     println("Running command:\n$command")
     run(command)
 
-    command = `gcc -m64 $C_FILE -o $E_FILE $SO_FILE $cflags $ldflags $ldlibs -Wl,-rpath,\$ORIGIN`
+    command = `$cc -m64 $C_FILE -o $E_FILE $SO_FILE $cflags $ldflags $ldlibs -Wl,-rpath,\$ORIGIN`
     println("Running command:\n$command")
     run(command)
 end

